src/web/screens/apiKeyManagement.tsx

```diff
@@ -35,6 +40,17 @@ function ApiKeyManagement() {
     }
   };
 
+  const onKeyEdit: OnApiKeyEdit = async (form, setApiKey, participantId?) => {
```

Comment:
is `participantId?` necessary here? I thought it would just be `participantId`

Reply:
Practically at this time this has very little relevance. I could just remove the participant id completely. 
`await onEdit(formData, setApiKey);`
I don't actually use it when I call the onEdit function. My reasoning for leaving it in is if in the future we wish to use the portal to manage keys as admin there's the ability to pass through the participant id.

Reply:
I'm thinking we should just get rid of `participantId` and change `EditApiKey` to not accept it, and instead just rely on the `'current'` participant. What do you think?

Reply:
I think getting rid of it from `OnApiKeyEdit` is good. Unsure if removing it from `EditApiKey` is good. It maintains the same pattern as all our other endpoints atm and it would help if in the future we wish to have admin edit any participants api keys for example.

Reply:
Let's get rid of it from `OnApiKeyEdit`. I don't feel too strongly either way about removing it from `EditApiKey` - there's a case that if it isn't used today or in the immediate future then we should remove it, but also you are right in that matches the pattern we usually use for web service endpoints.